src/containers/b2b/RequisitionPageMain.tsx

```diff
@@ -418,9 +418,10 @@ class RequisitionPageMain extends Component<RouteComponentProps<RequisitionPageM
         })
         .then((res) => {
           if (res.status === 200 || res.status === 201) {
-            this.setState({ selectedProducts: [], multiSelectMode: false, addToCartLoader: false });
+            this.setState({ selectedProducts: [], multiSelectMode: false });
             onCountChange(cartName, itemQuantity);
           }
+          this.setState({ addToCartLoader: false });
```

Comment:
does this trigger a second refresh of the components on the page if the previous one is run?

Reply:
@shaunmaharaj hmm... I think I need to put it in else block